server.js

```diff
@@ -1,67 +1,30 @@
 const express = require('express');
 const mongoose = require('mongoose');
-const bodyParser = require('body-parser');
-const {Schema} = mongoose;
-
 const app = express();
-const port = 3001;
-const jsonParser = bodyParser.json();
 
-const todosScheme = new Schema({
-    text: {
-        type: String,
-        required: true
-    },
-    color: {
-        type: String,
-        default: 'red'
-    },
-    checked: {
-        type: Boolean,
-        default: false
-    }
-});
-const Todos = mongoose.model('Todos', todosScheme);
+const router = require('./routes/router.js');
+app.use(router);
+
+// to .env /////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Use **// TODO: <thing that needs to be done>** kind of comments (they are being highlighted by the IDE.